.github/workflows/create_promptflow_release_tag.yml

```diff
@@ -0,0 +1,35 @@
+name: Create promptflow release tag
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create_release_tag:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: create branch
+        run: |
+          set -x -e
+
+          branch_name=$(git rev-parse --abbrev-ref HEAD)
+          echo "branch name: $branch_name"
+          release_version=${branch_name#release/promptflow/}
+          echo "release version: $release_version"
```

Comment:
Should Fail if release version doesn't match the pattern.

Also git rev-parse is getting SHA?

Reply:
latest commit have resolved these